.storybook/config.js

```diff
@@ -35,23 +35,23 @@ addDecorator(
     styles: stylesheet => {
       stylesheet.infoBody = {
         color: COLOR.TEAL.DARKEST,
-        fontFamily: 'Inter-UI-Regular',
+        fontFamily: 'Inter-Regular',
```

Comment:
So shouldn't this use the variables defined in `@teamleader/ui-typography`?

`--font-family-regular`, etc